components/brave_ads/core/internal/ads_impl.cc

```diff
@@ -93,6 +94,45 @@ void AdsImpl::Shutdown(ShutdownCallback callback) {
   std::move(callback).Run(/*success=*/true);
 }
 
+void AdsImpl::GetInternals(GetInternalsCallback callback) {
+  if (task_queue_.should_queue()) {
+    return task_queue_.Add(base::BindOnce(&AdsImpl::GetInternals,
+                                          weak_factory_.GetWeakPtr(),
+                                          std::move(callback)));
+  }
+
+  // TODO(tmancey): Decouple.
+  database::table::CreativeSetConversions database_table;
+  database_table.GetActive(base::BindOnce(&AdsImpl::GetActiveCallback,
+                                          weak_factory_.GetWeakPtr(),
+                                          std::move(callback)));
+}
+
+// TODO(tmancey): Decouple.
+void AdsImpl::GetActiveCallback(
+    GetInternalsCallback callback,
+    const bool success,
```

Comment:
As raised on slack today, `const` arguments are not allowed by the google guidelines.

Reply:
This is consistent with ads library code. Let's discuss separately and change this through the whole ads library.

Reply:
Well, if we know that this is not supposed to be done I don't see why we should introduce new cases of it, only to have it removed later on. I understand the argument of consistency, but the consistency that is more important is with chromium rules.

Reply:
I don't really want to do a discussion here, let's do it separately, if it works for you.

Reply:
Sure. It is not something too important.